### PR TITLE
chore: replace cs and staticanalysis with reusable workflows

### PR DESCRIPTION
**Pull request by bshaffer**
see https://github.com/GoogleCloudPlatform/php-tools/pull/148

### CODE REVIEW

File: .github/workflows/tests.yml [hunk 1]
```diff
@@ -43,38 +43,12 @@ jobs:
...
-              run: vendor/bin/phpcs -s --standard=./ruleset.xml
-
-    staticanalysis:
-      runs-on: ubuntu-latest
-      name: PHPStan Static Analysis
-      steps:
-      - uses: actions/checkout@v4
-      - name: Setup PHP
-        uses: shivammathur/setup-php@v2
+        uses: GoogleCloudPlatform/php-tools/.github/workflows/code-standards.yml@reusable-workflow
```
Comment (bshaffer) [inline .github/workflows/tests.yml:47]:
```suggestion
        uses: GoogleCloudPlatform/php-tools/.github/workflows/code-standards.yml@main
```

File: .github/workflows/tests.yml [hunk 2]
```diff
@@ -43,38 +43,12 @@ jobs:
...
-          command: composer install
-      - name: Run Script
-        run: |
-          vendor/bin/phpstan analyse
+          path: src
+
+
+    static-analysis:
+        name: PHPStan Static Analysis
+        uses: GoogleCloudPlatform/php-tools/.github/workflows/static-analysis.yml@reusable-workflow
```
Comment (bshaffer) [inline .github/workflows/tests.yml:54]:
```suggestion
        uses: GoogleCloudPlatform/php-tools/.github/workflows/static-analysis.yml@main
```

File: .github/workflows/tests.yml [hunk 3]
```diff
@@ -43,38 +43,12 @@ jobs:
...
-        with:
-          timeout_minutes: 10
-          max_attempts: 3
-          command: composer install
-      - name: Run Script
-        run: |
-          vendor/bin/phpstan analyse
+          path: src
+
+
```
Comment (bshaffer) [inline .github/workflows/tests.yml:51]:
```suggestion
```

File: .github/workflows/tests.yml [hunk 4]
```diff
@@ -43,39 +43,11 @@ jobs:
...
-      name: PHPStan Static Analysis
-      steps:
-      - uses: actions/checkout@v4
-      - name: Setup PHP
-        uses: shivammathur/setup-php@v2
-        with:
-          php-version: '8.2'
-      - name: Install Dependencies
-        uses: nick-invision/retry@v3
+        uses: GoogleCloudPlatform/php-tools/.github/workflows/code-standards.yml@main
```
Comment (Hectorhammett) [inline .github/workflows/tests.yml:47]: Niiiice!

**Review by Hectorhammett — APPROVED** [review]

**Pull request merged**